src/Frontend/Components/AttributionList/AttributionList.tsx

```diff
@@ -2,58 +2,139 @@
+import { setMultiSelectSelectedAttributionIds } from '../../state/actions/resource-actions/attribution-view-simple-actions';
+import { savePackageInfo } from '../../state/actions/resource-actions/save-actions';
+import { openPopup } from '../../state/actions/view-actions/view-actions';
 import { useAppDispatch, useAppSelector } from '../../state/hooks';
-import { getSelectedAttributionIdInAttributionView } from '../../state/selectors/attribution-view-resource-selectors';
-import { DisplayPackageInfos } from '../../types/types';
-import { convertPackageInfoToDisplayPackageInfo } from '../../util/convert-package-info';
-import { getAlphabeticalComparerForAttributions } from '../../util/get-alphabetical-comparer';
-import { AttributionCountsPanel } from '../AttributionCountsPanel/AttributionCountsPanel';
-import { FilterMultiSelect } from '../Filter/FilterMultiSelect';
-import { useFilteredAttributions } from '../Filter/FilterMultiSelect.util';
-import { IconButton } from '../IconButton/IconButton';
-import { PackageCard } from '../PackageCard/PackageCard';
-import { AttributionsViewPackageList } from '../PackageList/AttributionsViewPackageList';
-import { ResizableBox } from '../ResizableBox/ResizableBox';
-
-const classes = {
-  topElements: {
-    display: 'flex',
-    alignItems: 'center',
-    flexWrap: 'wrap',
-    justifyContent: 'space-between',
-  },
-  root: {
-    margin: '5px',
-    display: 'flex',
-    flexDirection: 'column',
-  },
-};
+import { getManualAttributions } from '../../state/selectors/all-views-resource-selectors';
+import {
+  getMultiSelectSelectedAttributionIds,
+  getSelectedAttributionIdInAttributionView,
+} from '../../state/selectors/attribution-view-resource-selectors';
+import { usePrevious } from '../../util/use-previous';
+import { useFilteredAttributions } from '../../web-workers/use-signals-worker';
+import { Checkbox } from '../Checkbox/Checkbox';
+import { List } from '../List/List';
+import { PACKAGE_CARD_HEIGHT, PackageCard } from '../PackageCard/PackageCard';
+import { SearchTextField } from '../SearchTextField/SearchTextField';
+import { SelectMenu } from '../SelectMenu/SelectMenu';
+import { Spinner } from '../Spinner/Spinner';
+import { ActionBar, ButtonGroup, Container } from './AttributionList.style';
+import {
+  getFilteredAndSortedPackageCardIdsAndDisplayPackageInfos,
+  useFilterMenuOptions,
+} from './AttributionList.util';
 
 export function AttributionList() {
   const dispatch = useAppDispatch();
   const selectedPackageCardIdInAttributionView = useAppSelector(
     getSelectedAttributionIdInAttributionView,
   );
+  const multiSelectSelectedAttributionIds = useAppSelector(
+    getMultiSelectSelectedAttributionIds,
+  );
+  const selectedAttributionIdAttributionView = useAppSelector(
+    getSelectedAttributionIdInAttributionView,
+  );
+  const manualAttributions = useAppSelector(getManualAttributions);
+
+  const [anchorEl, setAnchorEl] = useState<HTMLElement>();
```

Comment:
In general, we try to avoid abbreviations.
```suggestion
  const [anchorElement, setAnchorElement] = useState<HTMLElement>();
```

Reply:
"anchorEl" is an exception to the rule because it's a common variable name in React for the anchor element. you can also search the code base and you'll find many other hits for this usage.